sortLoot.py

```diff
@@ -8,41 +8,49 @@ def PlotRisk(): # goes through enemies potential attacks and damage according to
 def PlotDamage(): # goes through potential attacks and their damage to the enemy depending on his defense profile
     pass
    
-def SortLoot(loot): # sort the loot array into our different bags
-    weapons = []
-    defense = []
-    food = []
-    supplies = []
+        
     
-    for item in loot:
-        if('weapon' in item):
-            item.remove('weapon')
-            weapons.append([item[0]] + item[1])
+# lootDict = {'item1' : {'itemName' : 'MagicWand','itemType' : 'weapon', 'itemStats': [3, 2, 1] },
             
-        elif('defense' in item):
-            item.remove('defense')
-            defense.append([item[0]] + item[1])
-        elif('food' in item):
-            item.remove('food')
-            food.append([item[0]] + item[1])
-        elif('supplies' in item):
-            item.remove('supplies')
-            supplies.append(item)
-        
+#             'item2' : {'itemName' : 'gloves', 'itemType' : 'defense','itemStats': [1, 1, 2] }
+#             }
+
+    
+lootDict = {'MagicWand': ['weapon', 3, 2, 1],
+             'gloves' : ['defense', 1, 1, 2],
+             'banana' : ['food', 10, 5],
+             'tape' : ['supplies', 5]}
     
+         
+def SortLootDict(loot):

```

Comment:
You should use type annotation instead of including it in the function name
`def sortLoot(loot: dict)`

additionally, this function to return something. probably a dictionary
`def sortLoot(loot: dict) -> dict:`

Reply:
Do you want it to return a dictionary that contains those dictionaries then? I followed the original requirement, but am fine changing it.

Reply:
What was the original requirement? The function appears to do nothing, as nothing is returned.